Makefile

```diff
@@ -229,10 +229,25 @@ bundle: manifests kustomize operator-sdk ## Generate bundle manifests and metada
 	$(KUSTOMIZE) build config/manifests | $(OSDK) generate bundle $(BUNDLE_GEN_FLAGS)
 	$(OSDK) bundle validate ./bundle
 
+.PHONY: bundle-crd-only
+bundle-crd-only: manifests kustomize operator-sdk ## Generate bundle manifests for CRDs only and metadata, then validate generated files.
```

Comment:
Any reason to keep the standard targets with the controller? What is the purpose of the controller anyway?

Reply:
We are still working with the IBM team to decide if we want to ship the complete functionality of the recipe controller someday. Will keep the existing target for that future use-case.

As of now, we need a target that bundles only the crd.